### DISCUSSION TITLE
Provide a guide for serialization in Sveltekit

### DISCUSSION THREAD

**Discussion by JohannesPertl** [formfcw/directus-extension-flexible-editor#28] [Feature Request]
Something like your render view for Vue https://github.com/formfcw/tiptap-render-view would be nice for Sveltekit

**Comment by formfcw** [comment]
Actually, [tiptap-render-view](https://github.com/formfcw/tiptap-render-view) is a kind of toolkit. So you could copy the structure of [tiptap-render-view/vue](https://github.com/formfcw/tiptap-render-view/tree/main/vue) and translate it into a Svelte (or React or whatever) app. Maybe AI tools could help with the translation from Vue to Svelte code?

Want to contribute?

**Reply by JohannesPertl** [reply]
I'll look into it when I have the time :)
